evap/staff/templates/staff_course_comments_quick.html

```diff
@@ -0,0 +1,436 @@
+                        type: "POST",
+                        url: "{% url 'staff:course_comments_update_publish' %}",
+                        data: parameters,
+                        error: function(){ window.alert("{% trans 'The server is not responding.' %}"); }
+                    });
+
+                    if(action === "unreview") {
+                        this.active.removeAttr("data-review");
+                        this.updateState();
+                        return;
+                    }
+
+                    this.active.attr("data-review", action);
+                    this.updateState();
+
+                    this.slide("right", true);
+                    this.focusedButton = this.slider.find("[data-action=" + action + "]");
+                    this.focusedButton.focus();
+                    clearTimeout(this.buttonTimeout);
+                    this.buttonTimeout = setTimeout($.proxy(this.finishSlide, this), 200);
+                },
+
+                finishSlide: function() {
+                    this.updateState();
+                    this.focusedButton.blur();
+                },
+
+                repeat: function(action) {
+                    let reviewed = [];
+                    let unreviewed = [];
+
+                    this.answers.items.each(function() {
+                        let item = $(this);
+                        if(item.is("[data-review]")) {
+                            reviewed.push(item);
+                        } else {
+                            unreviewed.push(item);
+                        }
+                    });
+
+                    if(action !== "all" && unreviewed.length) {
+                        this.index = reviewed.length;
+                    } else {
+                        this.index = 0;
+                    }
+
+                    this.items = reviewed.concat(unreviewed);
+
+                    let direction = action === "start"? "right" : "left";
+                    this.answers.slide(this.active, direction);
+                    this.updateState();
+                },
+
+                updateState: function(noFooter) {
+                    this.slider.find(".slider-inner > :first-child").toggleClass("disabled", this.index === 0);
+                    this.slider.find(".slider-inner > :last-child").toggleClass("disabled", !this.isReviewing);
+
+                    this.slider.find("[data-counter=reviewed-left]").text(this.numReviewed);
+                    this.slider.find("[data-counter=unreviewed-left]").text(this.index - this.numReviewed);
+                    this.slider.find("[data-counter=reviewed-right]").text(this.numAlreadyReviewed);
+                    let unreviewed = this.items.length - this.index - this.numAlreadyReviewed - 1;
+                    this.slider.find("[data-counter=unreviewed-right]").text(Math.max(unreviewed, 0));
+
+                    if(!this.isReviewing) {
+                        let reviewed = this.numReviewed === this.index;
```

Comment:
i think it's a bit weird to use the index here, is there some kind of numTotal somewhere that you can use?